src/test/java/de/tum/in/www1/artemis/exercise/programmingexercise/ProgrammingExerciseGitlabJenkinsIntegrationTest.java

```diff
@@ -364,6 +364,21 @@ void configureRepository_testBadRequestError() throws Exception {
     @WithMockUser(username = TEST_PREFIX + "instructor1", roles = "INSTRUCTOR")
     void exportInstructorRepositories() throws Exception {
         programmingExerciseTestService.exportInstructorRepositories_shouldReturnFile();
+        // we export three repositories (template, solution, tests) and for each repository the temp directory should be deleted
```

Comment:
The indentation of `@Test` three lines above looks off

Reply:
I think that's the github view because in IntelliJ it looks fine.